modules/installation-restricted-network-samples.adoc

```diff
@@ -26,23 +43,32 @@ not addressed in this procedure.
 
 .Procedure
 
+. Access the images of a specific imagestream to mirror, for example:
++
+----
+$ oc get is <imagestream> -n openshift -o json | jq .spec.tags[].from.name | grep registry.redhat.io
+----
++
```

Comment:
Can you ask for a command that doesn't use `jq`? We're not supposed to add new `jq` commands because that package isn't available by default in RHEL.

Is there anything else a user needs to know about the imagestream name in this context?

Reply:
Because the restriction on using jq started after this PR was created, we should be able to let this one go, unless @bmcelvee you can find an example not using it.